doc/design/requirements.md

```diff
@@ -0,0 +1,142 @@
+# Requirements and goals
+
+Includes the overlapping projects of TiUp, TiUp Cluster, and TiUp Playground.
+
+We should distinguish between goals and requirements for the long term vs an initial release.
+
+## Vision and high-level goals
+
+* Help expand the TiDB ecosystem by making it easy to share, install, and deploy components
+* Manage and deploy TiDB and TiKV clusters
+* Open source project, with a community of users and contributors
+* Help TiDB and TiKV developers run and test their code in a production-like environment
+* Make TiDB an easy to use database
+* Be the standard way for TiDB users to manage their clusters
+* Make it easy to try out TiDB, on a single machine and a cluster
+
+## Use cases
+
+* A new user with no experience with TiDB can try out TiDB with minimal friction.
+* A developer of TiDB or TiKV can setup a local environment to test, benchmark, and debug their code.
+* An oncall DBA or developer can quickly setup a cluster matching one where a bug was reported.
+* A user reporting a bug can provide clear steps to reproduce based on a TiUp cluster configuration
+* A TiDB DBA can manage a production cluster, including TODO
```

Comment:
Is it WIP?

Reply:
I don't know all the requirements for tiup-cluster or what people do at the moment to manage their clusters, so I hope somebody else can help with this part.